### PR TITLE
Fixes and pinpointer update

### PR DESCRIPTION
**Pull request by Markolie**
Changes:
1) Refactors our pinpointer code so it's up-to-date with /tg/, and fixes the item tracker function on the advanced pinpointer. Replaces the PDA pinpointer in the uplink with the much more useful advanced pinpointer, as no-one ever bought the latter. Also fixes the pinpointer "off" sprite.
2) Fixes being unable to unbuckle anyone from pipes. Fixes https://github.com/ParadiseSS13/Paradise/issues/2286.
3) Fixes being able to sign up as a posibrain with the antugHUD active. Fixes https://github.com/ParadiseSS13/Paradise/issues/2283.
4) Cyborg shakers now only regenerate drinks. Fixes https://github.com/ParadiseSS13/Paradise/issues/2281.
5) Humanoid slowdowns are now added on top of wheelchair slowdowns. Not an ideal fix due to some slowdowns technically seen not applying, but there's not a lot of other options to prevent powergaming. Fixes https://github.com/ParadiseSS13/Paradise/issues/2280.
6) Fixes some incorrect flooring in the kitchen.
7) Removes some useless Animalize checks, and removes the ability for players to respawn as statues due to frequent abuse.
8) Fixes being unable to access the player panel of clientless mobs.
9) Radiation --> apply_effect in goon radiation power.
10) Fingerprints are now added when pipes are constructed.
11) Robots can now unbuckle people. Fixes https://github.com/ParadiseSS13/Paradise/issues/2293.
12) Fixes the drone welcome message. Fixes https://github.com/ParadiseSS13/Paradise/issues/2296.

### CODE REVIEW

**Comment by Fox-McCloud** [comment]
Yay.

File: code/_globalvars/lists/reagents.dm [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+// Rare chemicals
+var/list/rare_chemicals = list("minttoxin","nanites","xenomicrobes","adminordrazine")
+// Standard medicines
+var/list/standard_medicines = list("charcoal","toxin","cyanide","morphine","epinephrine","space_drugs","serotrotium","mutadone","mutagen","teporone","lexorin","silver_sulfadiazine","salbutamol","perfluorodecalin","omnizine","synaptizine","haloperidol","potass_iodide","pen_acid","mannitol","oculine","styptic_powder","methamphetamine","spaceacillin","carpotoxin","lsd","ethanol","ammonia","diethylamine","antihol","pancuronium","lipolicide","condensedcapsaicin","frostoil","amanitin","psilocybin","nothing","salglu_solution","neurotoxin")
```
Comment (Fox-McCloud) [inline code/_globalvars/lists/reagents.dm]: cyanide---standard medicine.

=p

Comment (Markolie) [inline code/_globalvars/lists/reagents.dm]: Yea, medicine is probably not the perfect name. Whatever :P.

Comment (Fox-McCloud) [inline code/_globalvars/lists/reagents.dm]: yeah, I don't really care, I just found it highly amusing.

**Comment by Belario** [comment]
Fixes are nice and all, but no more player statues?
Though they're very very OP, and I see the reasoning for doing so, taking them off the respawn list is a bit !!FUN!! killing.

**Comment by Markolie** [comment]
Though they're very very OP, and I see the reasoning for doing so, taking them off the respawn list is a bit !!FUN!! killing.
Well, you answered your own question there :P.

**Comment by Belario** [comment]
Ehhh... Maybe if they had some ridiculous weakness, or not have such high health...
Be a shame to see them go the way of the spider is all.

**Pull request merged**